### ISSUE TITLE
drush si <profile_name> gives error

### ISSUE THREAD

**Issue by robertgarrigos** [backdrop-contrib/drush#150]
`drush si <profile>` should install a site with the given profile, but it's not working. I'm trying, over lando:

`lando drush si my_profile --db-url=mysql://backdrop:backdrop@database/backdrop`

which gives the error:

`Sorry, the profile you have chosen cannot be loaded.fwrite() expects parameter 1 to be resource, string given in drush_print() (line 45 of phar:///usr/local/bin/drush/includes/output.inc).`

----

PR: https://github.com/backdrop-contrib/drush/pull/157

**Comment by kreynen** [comment]
Since we need this working, I'd take a look.  Does this only happen with a specific profile?  Is that profile public so I can test this myself?

**Comment by robertgarrigos** [comment]
It's not a public profile (yet). It's a profile I'm working on, but it works when installing thru web.

**Comment by serundeputy** [comment]
thanks @robertgarrigos and @kreynen 

I've filed a PR at https://github.com/backdrop-contrib/drush/pull/157

In my testing I able to install the `minimal` profile with:

```bash
lando drush si  minimal --db-url=mysql://backdrop:backdrop@database/backdrop
```

please give it a test and see if it works for you.

**Comment by robertgarrigos** [comment]
Thanks @serundeputy. How can I specify to use this pr instead of the stable drush lando uses?

**Comment by serundeputy** [comment]
https://github.com/backdrop-contrib/drush/wiki

Once you have that; use git to checkout the branch you want

**Comment by robertgarrigos** [comment]
ok thanks

**Comment by robertgarrigos** [comment]
I tried this pr. It looks like it's working, this is, Backdrop gets installed with any profile available (my own profile also). That's good.

However, I cannot login to Backdrop with the credentials  provided by the script. I tried by adding --account-name and --account-pass arguments and cannot login either. I guess this is a different issue.

Did you manage to login to Backdrop once installed?
